docs/network/process-exits.md

```diff
@@ -61,34 +61,45 @@ import { RootChain, OmgUtil } from "@omisego/omg-js";
 
 const web3 = new Web3(new Web3.providers.HttpProvider(web3_provider_url));
 const rootChain = new RootChain({ web3, plasmaContractAddress });
+
+// define constants
+const exitProcess = {
+  currency: "0xd92e713d051c37ebb2561803a3b5fbabc4962431",
+  maxExits: 1,
+  address: "0x8CB0DE6206f459812525F2BA043b14155C2230C0",
+  privateKey: "0xCD55F2A7C476306B27315C7986BC50BD81DB4130D4B5CFD49E3EAF9ED1EDE4F7"
+}
 ```
 
 > - `web3_provider_url` - the URL to a full Ethereum RPC node (local or from infrastructure provider, e.g. [Infura](https://infura.io/)).
 > - `plasmaContractAddress` - `CONTRACT_ADDRESS_PLASMA_FRAMEWORK` for defined [environment](/environments).
 
 ### 3. Process an exit
 
-> Exit processing is the same for both ETH and ERC20 UTXOs. This method demonstrates exit processing for ETH funds. If you want to process an ERC20 exit, change the `token` value to a corresponding smart contract address.
+> Exit processing is the same for both ETH and ERC20 UTXOs. This method demonstrates exit processing for ERC20 funds. If you want to process an ETH exit, change the `currency` value of the `exitProcess` object into `OmgUtil.transaction.ETH_CURRENCY`.
 
 Before you start processing, you can check the exit queue to see how many available exits the OMG Network has at a given moment:
 
 ```js
 async function processExit() {
   // get exit queue
   const ethQueue = await rootChain.getExitQueue();
-
-  // process exits
-  const exitReceipt = await rootChain.processExits({
-    token: OmgUtil.transaction.ETH_CURRENCY,
-    exitId: 0,
-    maxExitsToProcess: 5,
-    txOptions: {
-      privateKey: "0xCD55F2A7C476306B27315C7986BC50BD81DB4130D4B5CFD49E3EAF9ED1EDE4F7",
-      from: "0x8CB0DE6206f459812525F2BA043b14155C2230C0"
-    };
-  });
-
-  return exitReceipt;
+  
+  if (ethQueue.length) {
+    // process exits
+    const exitReceipt = await rootChain.processExits({
+      token: exitProcess.currency,
+      exitId: 0,
+      maxExitsToProcess: exitProcess.maxExits,
```

Comment:
should we provide an explanation as to why we use `exitId: 0` ?

Reply:
I actually don't remember why but it doesn't work otherwise : )